armtemplates/parameters.bicepparam

```diff
@@ -0,0 +1,16 @@
+using 'template.bicep' /*TODO: Provide a path to a bicep template*/
+
+param virtualMachines_iagora_name = 'iagora'
```

Comment:
Could we add a description for these parameters? It should make clear what this parameter is used for and what values we should provide.

Reply:
it's done !

Reply:
Great) Thanks) Just discovered that we could also use the `@description` tag for that, ex:
```bicep
@description('SSH public key for VM access.')
```
What about using this approach instead?

Reply:
yes, i agree !